routes/scripts/shared/fix_fullTxt.js

```diff
@@ -11,29 +12,42 @@ DB.conn.tx(t => {
     const batch = []
     // CONSOLIDATE ATTACHMENT FORMATTING
     pads.forEach(d => {
-      let fullTxt = `${d.title}\n\n`;
+      let fullTxt = d.title !== null ? `${d.title}\n\n` : '';
 
       d.sections.forEach(c => {
-        if (store_instructions && c.title) fullTxt += `${c.title}\n`;
-        if (store_instructions && c.lead) fullTxt += `${c.lead}\n`;
+        let currentTxt = '';
 
         c.items.forEach(b => {
           if (b.type === 'group' && b.items?.length > 0) {
             b.items.forEach(a => {
               a.forEach(z => {
                 const text = retrieveText(z);
-                fullTxt += `${text}\n`;
+                if (text !== null) {
+                  currentTxt += `${text}\n`;
+                }
               })
             })
           } else {
             const text = retrieveText(b);
-            fullTxt += `${text}\n`;
+            if (text !== null) {
+              currentTxt += `${text}\n`;
+            }
           }
         })
-      })
 
+        if (currentTxt.length) {
+          if (store_instructions && c.title) {
+            fullTxt += `${c.title}\n`;
+          }
+          if (store_instructions && c.lead) {
```

Comment:
hmm this is not only for templates so I would remove the store_instuctions conditions here

Reply:
not sure what you mean. that's your code :)

Reply:
I just moved it from here https://github.com/UNDP-Accelerator-Labs/platform/pull/285/files#diff-e38c85e90175768e57039d33c465662d62c108dc4749bf4298a86918c31960a1L18